chain-impl-mockchain/src/ledger.rs

```diff
@@ -582,9 +585,17 @@ fn internal_apply_transaction(
     witnesses: &[Witness],
     fee: Value,
 ) -> Result<Ledger, Error> {
-    assert!(inputs.len() < 255);
-    assert!(outputs.len() < 255);
-    assert!(witnesses.len() < 255);
+    if inputs.len() >= 255 {
```

Comment:
the doc says 256 inputs: https://github.com/input-output-hk/rust-cardano/blob/master/chain-impl-mockchain/doc/format.md#common-structure

Reply:
I think `< 255` is indeed 1 too short, pretty much a copy paste error